frontend/src/app/services/api.service.ts

```diff
@@ -132,4 +132,18 @@ export class ApiService {
   listPools$(interval: string | null) : Observable<PoolsStats> {
     return this.httpClient.get<PoolsStats>(this.apiBaseUrl + this.apiBasePath + `/api/v1/mining/pools/${interval}`);
   }
+
+  getPoolStats$(poolId: number, interval: string | null): Observable<PoolStat> {
+    return this.httpClient.get<PoolStat>(this.apiBaseUrl + this.apiBasePath + `/api/v1/mining/pool/${poolId}/${interval}`);
+  }
+
+  getPoolBlocks$(poolId: number, fromHeight: number): Observable<BlockExtended[]> {
+    if (fromHeight !== undefined) {
+      return this.httpClient.get<BlockExtended[]>(this.apiBaseUrl + this.apiBasePath +
+        `/api/v1/mining/pool/${poolId}/blocks/${fromHeight}`);
```

Comment:
Just use something like `fromHeight ? '/' + fromHeight : ''`

Reply:
Fixed in Fixed in https://github.com/mempool/mempool/pull/1242/commits/1e96c93557dafe2eef5460ceb23e5c1b09f3e741